packages/react-components/src/components/MessageThread.tsx

```diff
@@ -287,30 +297,42 @@ const memoizeAllMessages = memoizeFnAll(
 
     if (message.type === 'chat') {
       const payload: ChatMessagePayload = message.payload;
-      messageProps.messageContainerStyle = styles?.chatMessageContainer;
+
+      const myChatMessageStyle = styles?.myChatMessageContainer || defaultMyChatMessageContainer;
+      const chatMessageStyle = styles?.chatMessageContainer || defaultChatMessageContainer;
+      messageProps.messageContainerStyle = message.payload.mine ? myChatMessageStyle : chatMessageStyle;
 
       const chatMessageComponent =
         onRenderMessage === undefined
           ? defaultChatMessageRenderer(messageProps)
           : onRenderMessage(messageProps, defaultChatMessageRenderer);
 
       const personaOptions: IPersona = {
-        text: payload.senderDisplayName,
         hidePersonalDetails: true,
         size: PersonaSize.size32
       };
 
+      const myChatItemMessageStyle = styles?.myChatItemMessageContainer || defaultMyChatItemMessageContainer;
+      const chatItemMessageStyle = styles?.chatItemMessageContainer || defaultChatItemMessageContainer;
+
       return {
-        gutter: payload.mine ? (
-          ''
-        ) : onRenderAvatar ? (
-          onRenderAvatar(payload.senderId ?? '', personaOptions)
-        ) : (
-          <Persona {...personaOptions} />
-        ),
+        gutter: {
+          styles: {
+            position: 'relative',
+            float: 'left'
+          },
+          content: payload.mine ? (
+            ''
+          ) : onRenderAvatar ? (
+            onRenderAvatar(payload.senderId ?? '', personaOptions)
+          ) : (
+            <Persona {...personaOptions} />
+          )
+        },
         contentPosition: payload.mine ? 'end' : 'start',
         message: {
-          className: mergeStyles({ width: 'calc(100% - 6.25rem)' }),
+          className: mergeStyles({ width: 'calc(100% - 5rem)' }),
```

Comment:
nit: move into styles file and ensure these items `myChatItemMessageStyle` `chatItemMessageStyle` have it to have the styles in one place